apply/processor/gen.go

```diff
@@ -187,7 +187,16 @@ func (g *GenerateProcessor) ApplyRegistry(cluster *v2.Cluster) error {
 	if err != nil {
 		return err
 	}
-	err = runt.(*runtime.KubeadmRuntime).GenerateRegistryCert()
+	rt, ok := runt.(*runtime.KubeadmRuntime)
+	if !ok {
+		return fmt.Errorf("valid runtime")
+	}
+	err = rt.GenerateRegistryCert()
+	if err != nil {
+		return err
+	}
+	// sealer need to update registry cert on other nodes, otherwise docker pull or push will report error
```

Comment:
here i think ,could we send to all nodes whether it is master0 or not after generate registry cert

Reply:
agress with you

Reply:
ok，I'd like to ask how to realize it
send to master0  has been implemented in rt.GenerateRegistryCert()

Reply:
don't worry about this，how about we split it ? method "GenerateRegistryCert" only generate registry cert, and we use another method to do send .